### DISCUSSION TITLE
process.env behaviour

### DISCUSSION THREAD

**Discussion by yasinosman** [parcel-bundler/parcel#5649] [General]
Hey, i am trying to use .env file for my project built with parcel. What i am curious about is that does parcel detect changes in .env files after build ? Or should it be?

Think of this example: I build the project using a script like this in a computer:
`"build": "parcel build --no-content-hash --no-source-maps index.html --public-url ./",`
and i move the build folder to another computer which contains a different `.env` file. But parcel seems to get information from the first .env file. So i have to change .env file between two computers and rebuild every time.

I want to know if this is an expected behavior or not ?

**Comment by devongovett** [comment]
I believe this should be working in the latest Parcel 2 nightlies. What version are you using?

**Reply by yasinosman** [reply]
i am using 1.12.4. Here is my package.json:
```
{
    "version": "1.0.0",
    "main": "index.js",
    "scripts": {
        "test": "jest --watchAll",
        "dev-client": "parcel index.html",
        "build": "parcel build --no-content-hash --no-source-maps index.html --public-url ./",
        "clean": "rm -rf .cache && rm -rf dist && echo \"Cleaning completed\"",
        "build-apache": "parcel build --no-content-hash --no-source-maps index.html --public-url ./ && find /var/www/html/ -maxdepth 1 -type f -delete && mv dist/* /var/www/html && service apache2 restart && echo \"Apache deploy completed\"",
        "deploy-apache": "find /var/www/html/ -maxdepth 1 -type f -delete && mv dist/* /var/www/html && service apache2 restart && echo \"Apache deploy completed\""
    },
    "devDependencies": {
        "@types/jest": "^26.0.19",
        "jest": "^26.6.3",
        "parcel-bundler": "^1.12.4",
        "sass": "^1.32.4"
    },
    "browserslist": [
        "last 3 and_chr versions",
        "last 3 chrome versions",
        "last 3 opera versions",
        "last 3 ios_saf versions",
        "last 3 safari versions"
    ],
    "dependencies": {
        "axios": "^0.21.1",
        "redux": "^4.0.5"
    }
}
```
